.eslintignore

```diff
@@ -0,0 +1 @@
+lib/*.js
```

Comment:
You should fix your IDE to include trailing linefeeds.
.editorconfig is great for that